### PR TITLE
Add possibility to install keepalived from repo

### PR DESCRIPTION
**Pull request by roumano**
default install mode is from git ( as before)

But can be change via `keepalived_install_from: 'repo'`

### CODE REVIEW

File: templates/etc/systemd/system/keepalived.service.j2 [hunk 1]
```diff
@@ -1,3 +1,8 @@
+{% if keepalived_install_from == 'git' %}
+{% set keepalived_path = '/usr/local/sbin/' %}
+{% else %}
+{% set keepalived_path = '/usr/sbin/' %}
+{% endif %}
```
Comment (tersmitten) [inline templates/etc/systemd/system/keepalived.service.j2]: Maybe move this logic to `vars` (as `keepalived_install_prefix`)?

File: README.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ Set up the latest or a specific version of [Keepalived](http://www.keepalived.or
 * `keepalived_git_repo`: [default: `https://github.com/acassen/keepalived.git`]: Keepalived git repo
 * `keepalived_version`: [default: `v2.2.2`]: Keepalived version to install
 
+* `keepalived_install_from`: [optional, default `git`]: how to install keepalived (actually it's support git/repo)
```
Comment (tersmitten) [inline README.md]: Maybe call this `keepalived_install_method` (with values `native` and `git`)

Comment (roumano) [reply README.md]: Fixed

File: templates/etc/systemd/system/keepalived.service.j2 (hunk 1 above)
Comment (roumano) [reply templates/etc/systemd/system/keepalived.service.j2]: Fixed

**Comment by roumano** [comment]
Change done as you requested
and also :
- add forgotten git.yml playbook
- improve the install from native (aka via repo)

**Pull request closed without merging**